src/react_native_web/queries.cljs

```diff
@@ -44,10 +44,15 @@
  (fn [db [_ {id :hook-id} {:keys [key]}]]
    (get-in db [:extensions/store id key])))
 
+(re-frame/reg-sub
+ :store/all
+ (fn [db [_ {id :hook-id} _]]
```

Comment:
i believe only subscriptions inside extension has this parameter, so it should be something like `(get-in db [:extensions/store "Extension ID"])))`

Reply:
btw, i'm not sure why it works on deployed version, hm